cfgov/unprocessed/apps/filing-instruction-guide/css/main.less

```diff
@@ -209,6 +209,30 @@
   } );
 }
 
-#ctrl-f-modal h3 {
-  font-size: unit(16px / @base-font-size-px, em);
+#ctrl-f-modal {
+  h3 {
+    font-size: unit(16px / @base-font-size-px, em);
+  }
+  svg {
+    .cf-icon-svg;
```

Comment:
I would add `()` to all mixins, making this `.cf-icon-svg()`. Same comment for the items below. See how we do that in other places in the codebase, such as https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/unprocessed/apps/form-explainer/css/form-explainer.less#L33

I believe the thinking is you could then search for `.cf-icon-svg()` and find all occurrences that are strictly mixed into another style, as opposed to being in markup or as part of a selector. Also, `()` implies it invokes and returns something else in its own place, which is what it is doing.